sdk/identity/Azure.Identity/src/Credentials/DefaultAzureCredentialOptions.cs

```diff
@@ -183,6 +184,8 @@ public string VisualStudioCodeTenantId
         /// the cache, the SharedTokenCacheCredential won't be used for authentication.
         /// Defaults to the value of environment variable <c>AZURE_USERNAME</c>.
         /// </remarks>
+        [Obsolete("SharedTokenCacheCredential is deprecated. For brokered authentication, consider using InteractiveBrowserCredential.")]
```

Comment:
We don't necessarily know the user is interested in brokered auth. Something like the following would be better.

```suggestion
        [Obsolete("SharedTokenCacheCredential is deprecated in favor of VisualStudioCredential.")]
```